app/frontend/components/_sign-table.scss

```diff
@@ -0,0 +1,57 @@
+.sign-table-row {
```

Comment:
Throughout this component, `px` are used, when `rem` is probably preferable here. We try and use rem fairly globally, since rems are a relative unit that can scale up and down in line with the user's zoom preferences, usually without breaking layouts. 

If you have a particular pixel measurement you can use, first of all, if it's margin or padding related, see if you can use a multiplier of `$global-margin` and `$global-padding` - these are variables that are already defined as rem, and allows us to modify the spacing globally. If it's something else, Foundation provides a helper function, `rem-calc(px)`, which will convert a pixel measurement into rem for you.